packages/flutter/lib/src/material/time_picker.dart

```diff
@@ -2103,21 +2104,7 @@ class _TimePickerDialogState extends State<TimePickerDialog> with RestorationMix
     final ShapeBorder shape = TimePickerTheme.of(context).shape ?? _kDefaultShape;
     final Orientation orientation = media.orientation;
 
-    final Widget actions = Row(
-      children: <Widget>[
-        const SizedBox(width: 10.0),
-        IconButton(
-          color: TimePickerTheme.of(context).entryModeIconColor ?? theme.colorScheme.onSurface.withOpacity(
-            theme.colorScheme.brightness == Brightness.dark ? 1.0 : 0.6,
-          ),
-          onPressed: _handleEntryModeToggle,
-          icon: Icon(_entryMode.value == TimePickerEntryMode.dial ? Icons.keyboard : Icons.access_time),
-          tooltip: _entryMode.value == TimePickerEntryMode.dial
-              ? MaterialLocalizations.of(context).inputTimeModeButtonLabel
-              : MaterialLocalizations.of(context).dialModeButtonLabel,
-        ),
-        Expanded(
-          child: Container(
+    final Widget actions = Container(
```

Comment:
We don't want to just remove the button here. We would only want to remove the button if the app asked for a specific mode that doesn't require the button.